maat-court-data-api/src/main/java/gov/uk/courtdata/prosecutionconcluded/service/ProsecutionConcludedDAO.java

```diff
@@ -0,0 +1,51 @@
+package gov.uk.courtdata.prosecutionconcluded.service;
+
+import gov.uk.courtdata.entity.RepOrderEntity;
+import gov.uk.courtdata.prosecutionconcluded.dto.ConcludedDTO;
+import gov.uk.courtdata.prosecutionconcluded.helper.CrownCourtCodeHelper;
+import gov.uk.courtdata.prosecutionconcluded.helper.ProcessSentencingHelper;
+import gov.uk.courtdata.prosecutionconcluded.helper.ResultCodeHelper;
+import gov.uk.courtdata.repository.CrownCourtStoredProcedureRepository;
+import gov.uk.courtdata.repository.RepOrderRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class ProsecutionConcludedDAO {
```

Comment:
Why is this Impl now? I don't think we are introducing DAO concept within MAAT API so far. We do service-> impl -> repository